C/other/Number_of_duplicates_using_Binary_search.c

```diff
@@ -0,0 +1,129 @@
+/* To print the occurrence of the given number in the list/array using Binary Search Technique */
+
+#include <stdio.h>
+int Binary_search_algo_for__finding_occurrence(int array[], int sizeOf_array, int number, int found)
+{
+int start = 0, end = sizeOf_array - 1;
+
+// initializing value
+int value = -1;
+
+while (start <= end)
+{
+int middle = (start + end)/2;
+
+/*** CASE : number equals middle element ***/
+if (number == array[middle]){  // if both are equal then we update the 'value'
+
+value = middle;
+if (found){ // searching in left side
+end = middle - 1;
+}
+else{ // searching in right side
+start = middle + 1;
+}
+
+}
+
+/*** CASE : number less than middle element ***/
+else if (number < array[middle]){
+end = middle - 1;
+}
+
+/*** CASE : number greater than middle element (last case) ***/
+else{
+start = middle + 1;
+}
+}
+return value;
+}
+int quickSort(int array[] , int lower_bound ,int  upper_bound)
+{
+    int temp ;
+     if(lower_bound<upper_bound)
+     {
+         temp = partition(array , lower_bound , upper_bound );// as partition function would return end index which would be used further
+        quickSort(array , lower_bound , temp-1);
+         quickSort(array ,temp+1, upper_bound);
+
+     }
+}
+
+int partition(int array[],int lower_bound,int upper_bound)
```

Comment:
Please put this function either before `quickSort` or add the declaration of this function just below the header files otherwise it'll throw error.

Reply:
@Kajol-Kumari I have done the required changes as mentioned. Could you please review it?